AppConfig.java

```diff
@@ -0,0 +1,9 @@
+package com.example.demo;
+import org.springframework.context.annotation.*;
+@Configuration
+public class AppConfig {
+	@Bean(initMethod = "init")
```

Comment:
@Bean is helpful when you have don't have access to class or you want to create lot of custom beans of the same class using different parameters. If you are using the default option - simply annotating the class with @Service / @Component will allow spring to automatically do this for you.